cluster/src/test/scala/org/apache/spark/sql/store/BugTest.scala

```diff
@@ -770,4 +772,50 @@ class BugTest extends SnappyFunSuite with BeforeAndAfterAll {
     snappy.sql("drop table testLimit")
     TestUtil.stopNetServer()
   }
+
+  test("support for 'default' schema without explicit quotes") {
+    val session = snc.snappySession
+    val serverHostPort = TestUtil.startNetServer()
+    val conn = DriverManager.getConnection(
+      "jdbc:snappydata://" + serverHostPort)
+
+    session.sql("create table default.t1(id bigint primary key, name varchar(10))")
+    var keys = session.sessionCatalog.getKeyColumns("default.t1")
+    assert(keys.length === 1)
+    assert(keys.head.toString === new Column("ID", null, "bigint", false, false, false).toString)
+
+    // also test from JDBC
+    val stmt = conn.createStatement()
+    stmt.execute("create table default.t2(id bigint not null primary key, name varchar(10))")
+    keys = session.sessionCatalog.getKeyColumns("default.t2")
+    assert(keys.length === 1)
+    assert(keys.head.toString === new Column("ID", null, "bigint", false, false, false).toString)
+
+    session.sql("insert into default.t1 values (1, 'name1'), (2, 'name2')")
+    var res = session.sql("select * from default.t1 order by id").collect()
+    assert(res === Array(Row(1L, "name1"), Row(2L, "name2")))
+    res = session.sql("select * from default.t1 where id = 1").collect()
+    assert(res === Array(Row(1L, "name1")))
+    res = session.sql("select * from `DEFAULT`.t1 where id = 2").collect()
+    assert(res === Array(Row(2L, "name2")))
+
+    stmt.execute("insert into default.t2 values (1, 'name1'), (2, 'name2')")
```

Comment:
Can you please add one insert with the table name itself in the backticks and see if that works too.
Like even this should work.

insert into "backtick"default"backtick"."backtick"t1"backtick" values (1, 'name1'), (2, 'name2')

Not able to format with backtick so wrote backtick with quotes.

Reply:
Ok. Btw, you can add code in triple backticks and also language i.e. ```sql and then end with triple backticks like below ("Edit" from options to see the formatting).

```sql
insert into `default`.`t1` values (1, 'name1'), (2, 'name2')
```

Reply:
ok. Thanks. I did try the triple backtick but may be I goofed up something.